### PR TITLE
Merge in fixes from upstream

### PR DESCRIPTION
**Pull request by dbluhm**
Merge in a more suitable fix from upstream for signing a request to the ledger when no public DID is available during send-proposal.

### CODE REVIEW

File: AdminAPI.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# ACA-Py Administration API
+
+## Using the OpenAPI (Swagger) Interface
+
+ACA-Py provides an OpenAPI-documented REST interface for adminstering the agent's internal state and sparking communication with connected agents.
```
Comment (burdettadam) [inline AdminAPI.md:5]: *administering

Comment (dbluhm) [reply AdminAPI.md:5]: That's a typo from upstream, unfortunately lol

File: AdminAPI.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+ * click on the Try it out button;
+ * fill in any data necessary to run the command;
+ * click Execute;
+ * check the response to see if the request worked as expected.
+
+The mechanical steps are easy, it’s fourth step from the list above that can be tricky. Supplying the right data and, where JSON is involved, getting the syntax correct - braces and quotes can be a pain. When steps don’t work, start your debugging by looking at your JSON. You may also choose to use a REST client like Postman or Insomnia which will provide syntax highlighting and other features to simplify the process.
+
+Because API methods will often kick off asynchronous processes, the JSON response provided by an endpoint is not always sufficient to determine the next action. To handle this situation as well as events triggered due to external inputs (such as new connection requests), it is necessary to implement a webhook processor, as detailed in the next section.
+
+The combination of an OpenAPI client and webhook processor is referred to as an ACA-Py Controller and is the recommended method to define custom behaviours for your ACA-Py-based agent application.
```
Comment (burdettadam) [inline AdminAPI.md:24]: *behaviors

File: AdminAPI.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+## Administration API Webhooks
+
+When ACA-Py is started with the `--webhook-url {URL}` command line parameter, state-management records are sent to the provided URL via POST requests whenever a record is created or its `state` property is updated.
+
+When a webhook is dispatched, the record `topic` is appended as a path component to the URL, for example: `https://webhook.host.example` becomes `https://webhook.host.example/connections` when a connection record is updated. A POST request is made to the resulting URL with the body of the request comprised by a serialized JSON object. The full set of properties of the current set of webhook payloads are listed below. Note that empty (null-value) properties are omitted.
```
Comment (burdettadam) [inline AdminAPI.md:30]: *comprised **of** a serialized

**Review by burdettadam — DISMISSED** [review]
I found some grammar improvements.

**Pull request merged**